docs/ispc.rst

```diff
@@ -663,6 +663,16 @@ To generate a text assembly file, pass ``--emit-asm``:
 To generate LLVM bitcode, use the ``--emit-llvm`` flag.
 To generate LLVM bitcode in textual form, use the ``--emit-llvm-text`` flag.
 
+To generate a stub after running the preprocessor, use the ``-E`` flag.
+
+::
+
+    ispc foo.ispc -E -o foo.i  OR  ispc foo.ispc -E -o foo.ispi
+
+In this mode, the preprocessor will assume ``stdout`` if no output file is
+specified.
+Both ``.i`` or ``.ispi`` are valid suffixes.
+
```

Comment:
```suggestion
To run only the preprocessor, use ``-E`` flag. By default the output will be directed to ``stdout``, otherwise to the file specified by ``-o`` flag. The output file with preprocessed source is assumed to have ``.i`` or ``.ispi`` extension.

::

    ispc foo.ispc -E -o foo.i
    ispc foo.ispc -E -o foo.ispi

```